python/tvm/relay/backend/contrib/ethosu/tir_to_cs_translator.py

```diff
@@ -105,22 +195,27 @@ def translate(tir_module, params):
     base_addresses : List[util.BaseAddress]
         base addresses to be used by the driver
     """
-
+    candidate_regions_for_scratch = [5, 2, 1]
```

Comment:
What is the meaning of these numbers? Maybe a comment would help there...

Reply:
Good point! I ll add a comment, in a follow up -- if thats alright ?

The comment would be something to describe the following point :

We have regions from 0 to 5 and we already are using regions 0, 3, and 4 statically for constants, input and output, respectively. Therefore, it leaves us with three more regions : 5, 2 and 1 to be used by memory pools.

Reply:
Thanks for the clarification, makes sense! Yeah, it makes sense to do it in the follow-up since the CI is green...

Reply:
@ekalda , I did the change as this PR got conflicted. See if it looks good now.

Reply:
Thanks for adding comment (and the other docstrings as well)! I suppose "regions ranging from 0-6" can be interpreted as both, total of 6 or 7 regions, depending on the counting philosophy :D I think it is clear enough what is meant there though.